lib/inputs/binaries/static/index.ts

```diff
@@ -1 +1,21 @@
-// TODO
+import { ExtractAction, ExtractedLayers } from "../../../extractor/types";
+import { streamToHash } from "../../../stream-utils";
+
+export const getNodeBinariesFileContentAction: ExtractAction = {
+  actionName: "node",
+  fileNamePattern: "**/*/node",
+  callback: streamToHash,
+};
+
+export function getBinariesHashes(extractedLayers: ExtractedLayers): string[] {
+  const hashes: string[] = [];
```

Comment:
is it an array or a map? looks like you use it as a map

Reply:
it is an array, being used like a map - that's a bug :)